var/spack/repos/builtin/packages/py-pybedtools/package.py

```diff
@@ -33,3 +34,6 @@ class PyPybedtools(PythonPackage):
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-pysam", type=("build", "run"))
     depends_on("py-six", type=("build", "run"))
+
+    def patch(self):
+        python('setup.py', 'cythonize')
```

Comment:
Direct invocation of setup.py is [deprecated](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html). Do the PyPI tarballs come pre-cythonized?

Reply:
no

Reply:
@pradyunsg I assume there's no other way to build this package without running `python setup.py cythonize` first?

Reply:
Actually, I might have spoken too soon, let me double check something

Reply:
So, the PyPI tarballs do come pre-cythonized, but the Github tarballs that this package was written with don't. How would you like me to handle this?

Reply:
We should always prefer using the PyPI tarballs over the GitHub tarballs: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Can you switch from `url` to `pypi` and run `spack checksum` to recompute all the checksums?

Reply:
Done